src/files/3_AllProjectsAnalyzers.editorconfig

```diff
@@ -0,0 +1,542 @@
+dotnet_diagnostic.CA1507.severity = none
+
+# CA1508: Avoid dead conditional code
+# Disabled because provides false positives in cases with double checked locking
+dotnet_diagnostic.CA1508.severity = none
+
+# CA1510: Use ArgumentNullException throw helper (ArgumentNullException.ThrowIfNull)
+dotnet_diagnostic.CA1510.severity = suggestion
+
+# CA1707: Identifiers should not contain underscores
+# Disabled because this is a common practice in test method names (and other cases)
+dotnet_diagnostic.CA1707.severity = none
+
+#CA1710: Identifiers should have correct suffix
+# Disabled because it's simply annoying (streams types must end with 'Stream', same for queue, stack, collection, dictionary, etc.)
+dotnet_diagnostic.CA1710.severity = none
+
+# CA1711: Identifiers should not have incorrect suffix
+# Disabled because it's sometimes OK to name something ending with Collection even if it doesn't implement ICollection
+dotnet_diagnostic.CA1711.severity = none
+
+# CA1716: Identifiers should not match keywords
+# Some method names such as "Get" or "Set" should be allowed
+dotnet_diagnostic.CA1716.severity = none
+
+# CA1720: Identifiers should not contain type names
+dotnet_diagnostic.CA1720.severity = none
+
+# CA1724: Type names should not match namespaces
+# Disabled because this is usually recommended for library authors
+dotnet_diagnostic.CA1724.severity = none
+
+# CA1805: Do not initialize unnecessarily
+dotnet_diagnostic.CA1805.severity = none
+
+# CA1812: Allow unreferenced code to exist
+dotnet_diagnostic.CA1812.severity = none
+
+# CA1819: Allow properties to return arrays
+dotnet_diagnostic.CA1819.severity = none
+
+# CA1822: Mark members as static
+# Disabled because the compiler might not always be aware of how a member is used
+dotnet_diagnostic.CA1822.severity = none
+
+# CA1848: Use the LoggerMessage delegates
+# Disabled because high-performance logging is overkill in most cases (except in libs with hot paths)
+dotnet_diagnostic.CA1848.severity = none
+
+# CA1859: Use concrete types when possible for improved performance
+# Disabled, down-casting doesn't degrade performance that much
+dotnet_diagnostic.CA1859.severity = none
+
+# CA1863: Use 'CompositeFormat'
+# Disabled, this is faily new and the performance gains over string.Format isn't worth it
+dotnet_diagnostic.CA1863.severity = none
+
+# CA2000: Dispose objects before losing scope
+# Disabled because disposables don't always need to be disposed in the current scope (ex: HttpClients provided by IHttpClientFactory)
+dotnet_diagnostic.CA2000.severity = none
+
+# CA2007: Do not directly await a Task
+# Disabled because using ConfigureAwait(true) isn't necessary in ASP.NET Core as there is no synchronization context
+dotnet_diagnostic.CA2007.severity = none
+
+# CA2008: Do not create tasks without passing a TaskScheduler
+# Disabled because it's OK to use the default scheduler in most applications (for desktop ones, this might be different)
```

Comment:
I can't remember where, but here's an example. To think I disabled it to prevent someone from doing the wrong fix. Maybe we could look at it in the future

![image](https://github.com/gsoft-inc/wl-dotnet-codingstandards/assets/14242083/3a2b2deb-8aed-47fc-ba09-e5e3e56d4917)